includes/forms/template.php

```diff
@@ -897,7 +897,8 @@ function give_default_cc_address_fields( $form_id ) {
 	$logged_in = is_user_logged_in();
 
 	if ( $logged_in ) {
```

Comment:
@raftaar1191 please remove this because it will generate unnecessary queries

Reply:
updated PR